CHANGELOG.md

```diff
@@ -10,12 +10,277 @@ New features:
 
 Bugfixes:
 
-* Only include direct dependencies in the output for `purs graph` (#3993, @colinwahl)
+Other improvements:
 
-Fixes a bug where the transitive closure of a module's dependencies
-where included in the `depends` field in the output for `purs graph`.
+## v0.14.0
 
-Other improvements:
+### Polykinds
+
+Polymorphic kinds, based on the [Kind Inference for Datatypes](https://richarde.dev/papers/2020/kind-inference/kind-inference.pdf) paper (#3779, #3831, #3929, @natefaubion)
+
+Kinds are to types what types are to terms (although kinds are themselves types now), but whereas we have polymorphic types, kinds were monomorphic.
+
+This meant that we were not able to abstract over kinds, leading for instance to a proliferation of proxy types:
+
+```purescript
+data Proxy (a :: Type) = Proxy
+data SProxy (a :: Symbol) = SProxy
+data RProxy (row :: # Type) = RProxy
+```
+
+Now we can have a single proxy type, whose parameter has a polymorphic kind.
+
+#### Type In Type
+
+The type-checker now supports `TypeInType` (or `Type :: Type`), so the old `Kind` data type and namespace is now gone. Kinds and types are the same and exist in the same namespace.
+
+Previously one could do:
+
+```purescript
+foreign import kind Boolean
+foreign import data True :: Boolean
+foreign import data False :: Boolean
+```
+
+Where the kind `Boolean` and type `Boolean` were two different things. This is no longer the case. The `Prim` kind `Boolean` is now removed, and you can just use `Prim` type `Boolean` in the same way. This is a breaking change.
```

Comment:
Can we add a note that we still need the `foreign import data`s because constructors are not automatically lifted? (i.e. no `DataKinds`)

Reply:
I added a note about this after the announce of the deprecation of `foreign import kind` delcarations.